examples/mlmc.cpp

```diff
@@ -160,6 +143,19 @@ int main(int argc, char* argv[])
     bool coarse_components = true;
     args.AddOption(&coarse_components, "-coarse-comp", "--coarse-components", "-no-coarse-comp",
                    "--no-coarse-components", "Store trace, bubble components of coarse M.");
+    const char* sampler_type = "simple";
+    args.AddOption(&sampler_type, "--sampler-type", "--sampler-type",
+                   "Which sampler to use for coefficient: simple, pde");
+    double kappa = 0.001;
+    args.AddOption(&kappa, "--kappa", "--kappa",
+                   "Correlation length for Gaussian samples.");
+    int num_samples = 3;
+    args.AddOption(&num_samples, "--num-samples", "--num-samples",
+                   "Number of samples to draw and simulate.");
+    bool blank_perm = false;
+    args.AddOption(&blank_perm, "--blank-perm", "--blank-perm",
+                   "--no-blank-perm", "--no-blank-perm",
+                   "Instead of loading SPE10 data, use constant permeability.");
     args.Parse();
     if (!args.Good())
     {
```

Comment:
`MPI_Finalize();` is incorrect in this block.  The `mpi_session` from above takes care of it.